arvados-s3/arvados-s3.py

```diff
@@ -0,0 +1,59 @@
+import json
```

Comment:
do we need test this pipeline with S3 remote option  where pipeline would download the dataset in one of the step.

For non-remote dataset where DKube downloads it , this pipeline does nothing specific to arvados-s3. Do we really need this pipeline in that case ?

Reply:
@mak-454